Parsers/ASimFileEvent/test/GoogleWorkspace_vimFileEvent_SchemaTest.csv

```diff
@@ -0,0 +1,92 @@
+﻿Result
+"(0) Error: Missing mandatory field [Type]"
```

Comment:
Why this error coming ? Need test case correction ?

Reply:
@anki-narravula 
strange, I re-run test today. It did not appear. I think we are good.
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/d1a8b0b3-0794-43b5-acd8-04d4ab0d11a9)

Reply:
also, updated results in PR